src/main/java/newbank/server/NewBank.java

```diff
@@ -75,8 +77,9 @@ private void validateUsername(final String username) throws UsernameInvalidExcep
   // Simple algorithm to check that the password meets the security requirements
   // must be non-empty
   private void validatePassword(final String password) throws PasswordInvalidException {
-    if (password.isEmpty()) {
-      throw new PasswordInvalidException();
+    if (!password.matches("^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d).+$")){
+        if (password.isEmpty());
```

Comment:
Hey Claudia! :wave: I don't think this if statement does anything, because of the semicolon that terminates it.

Reply:
I think the two conditions could be in the same `if`, connected with an OR.

Reply:
Thank you for checking! I'll update :)

Reply:
I've made the change but come to think of it... I could just remove 'isEmpty' since the first if statement will do that check anyways (will not accept empty string as needs at least one number etc.)

Reply:
Your second comment has only just appeared "I think the two conditions could be in the same if, connected with an OR." - I'll test using OR and removing the isempty completely